tonic/src/transport/service/router.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 use tower_service::Service;
 
 #[derive(Debug)]
-pub(crate) struct Routes<A, B, Request> {
+pub struct Routes<A, B, Request> {
```

Comment:
Note to self: Check if this really needs to be `pub`.

Reply:
It does unfortunately since it needs to appear in trait bounds. I've made it `#[doc(hidden)]`.